app/employee-app/backend/modules/authorization/authorization.bal

```diff
@@ -0,0 +1,75 @@
+//
+// Copyright (c) 2005-2024, WSO2 LLC.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+// 
+// http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+// 
+import ballerina/http;
```

Comment:
```suggestion

import ballerina/http;
```

Reply:
Do we need a extra line in here? None of the other files contain an extra line like this.

Reply:
Okay. Btw common practice is like this. Instead of adding an additional comment line in [#L17](https://github.com/wso2/people-ops-suite/blob/3eaf2147f0e20b57edcc7f95a23ad5e9f0721307/app/employee-app/backend/modules/authorization/authorization.bal#L17) you can seperate the code from the license header just by adding a new line.

Reply:
@Nuvindu It seems the formatter automatically removes the extra line(line 17 in your requested change)